feat: rename aliases to registryAliases (#15877)
renovatebot/renovate: docs/usage/configuration-options.md

For more details and examples, see our [documentation for the `regex` manager](/modules/manager/regex/).
For template fields, use the triple brace `{{{ }}}` notation to avoid Handlebars escaping any special characters.

## registryAliases

> The `registryAliases` object is used for configuring registry registryAliases.

Reviewer: ```suggestion
You can use the `registryAliases` object to set registry aliases.
```

> Currently it is needed/supported for the `helm-requirements`, `helmv3` and `helmfile` managers only.

Reviewer: ```suggestion
This feature only works with these managers:

- `helm-requirements`
- `helmv3`
- `helmfile`
```

> The above managers include this default registryAlias:

Reviewer: ```suggestion
The managers listed above all have this default registryAlias:
```
